dpk_connector/README.md

```diff
@@ -0,0 +1,29 @@
+# DPK Connector
+
+DPK Connector is a scalable and compliant web crawler developed for data acquisition towards LLM development. It is built on [Scrapy](https://scrapy.org/).
+
```

Comment:
Can you link to the overview.md?  As is, I can't tell if this is a library, a main() or how to use it.

Reply:
Added the link. `How to use` also has a link to it.